telugu/Telanganatelugu.md

```diff
@@ -0,0 +1 @@
+బతుకమ్మ - తెలంగాణ నేల పూల కవాతు అలికి ముగ్గులు పెట్టిన లోగిళ్ళు సింగారించుకున్న చెల్లెల్ల చేతి నైపుణ్యం వెలిసిన పూల పిరమిడ్ల ఇంధ్ర ధనువులు'' (నేల మీది చందమామలు, కాసుల లింగారెడ్డి) అని బతుకమ్మను తెలంగాణ సాంస్కృతిక చిహ్నంగా చేసుకుని నెత్తిన ఎత్తుకోవడంలో మిగతా ప్రాంతాల నుంచి వేరుపడి తన ప్రత్యేక అస్తిత్వాన్ని చాటుకోవడమే. ఇదంతా కవి అంతర్గత, బాహ్య వలసాధిపత్యాలను వ్యతిరేకించే క్రమంలో చేసే కవితా గానమే.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.